changelog.json

```diff
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "2.2.4": [
+      "[New] Warn when committing to a protected branch - #7023",
+      "[Added] Adding integration for Xcode as external editor - #8255. Thanks @msztech!",
+      "[Fixed] Changed wording to match git autocrlf behavior - #5741. Thanks @jmhammock!",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Fixed] Fix arrow key navigation of Changes and History lists (#6845) - #6845. Thanks @JMcKiern!",
+      "[Fixed] Handle multibyte characters in names - #8534",
+      "[Fixed] Keep PR badge on top of progress bar - #8622",
+      "[Fixed] Make newly cloned repositories load pull requests correctly - #8745",
```

Comment:
```suggestion
```

This was a bug that was only present on beta (it was introduced by #8635) so I don't think it should be part of the release notes for prod.